conda_smithy/licenses.txt

```diff
@@ -185,12 +220,25 @@ Glide
 Glulxe
 GLWTPL
 gnuplot
+GPL-1.0
+GPL-1.0+
 GPL-1.0-only
 GPL-1.0-or-later
+GPL-2.0
```

Comment:
This is deprecated. We didn't add any deprecated licenses here.

Reply:
Should I then only include licenses that are not deprecated (and exclude the ones from here that are now deprecated)  or all licenses that are currently here + all not deprecated licenses?